arch/lkl/Kconfig

```diff
@@ -10,26 +10,6 @@ config LKL
        select NO_HZ_IDLE
        select NO_PREEMPT
        select ARCH_WANT_FRAME_POINTERS
-       select NET
-       select MULTIUSER
-       select INET
```

Comment:
Both commits look reasonable to me. However, the removals here will break `tools/lkl` compilation without `CONFIG_NET=y`, etc. If this does get merged as-is then I don't think it should be closing https://github.com/lkl/linux/issues/541 .

Reply:
Oh, I re-read the description of issue #541, do you mean that you expect `lib/net.c` to be compiled conditionally according to the config options?

If so, I missed this point before, thanks for the clarification.

Reply:
I think this is an issue that can be fixed independently and won't prevent this PR from being merged. So I have removed the `Fixes` tags from the commit message and PR description for now.

Reply:
> Oh, I re-read the description of issue https://github.com/lkl/linux/issues/541, do you mean that you expect lib/net.c to be compiled conditionally according to the config options?

Exactly.

> If so, I missed this point before, thanks for the clarification.

No worries! FWIW, I'd still be fine with this as-is, but just want to make sure that the `tools/lkl` breakage remains tracked.